doc/howto/instances_create.md

```diff
@@ -72,6 +72,19 @@ To start an instance, send a PUT request to change the instance state:
 
     lxc query --request PUT /1.0/instances/<instance_name>/state --data '{"action": "start"}'
 
+If you would like to start the instance upon creation, set the `start` property to true. The following example will create the container, then start it:
+
+    lxc query --request POST /1.0/instances --data '{
+      "name": "<instance_name>",
+      "source": {
+        "alias": "<image_alias>",
+        "protocol": "simplestreams",
+        "server": "<server_URL>",
+        "type": "image"
+      }
+      "start": true
+    }'
+
 See {ref}`instances-manage-start` for more information.
```

Comment:
This link is about the command for starting the instance, so it should go above your addition.

Reply:
Ah, thanks. Moved the link back to original location.